config/locales/application_form.yml

```diff
@@ -962,6 +962,10 @@ en:
           attributes:
             ethnic_group:
               blank: Choose your ethnic group
+        candidate_interface/training_with_a_disability_form:
+          attributes:
+            disclose_disability:
+              inclusion: Choose if you want to ask for help to become a teacher
```

Comment:
Any idea why `training_with_a_disability_form` validates on `inclusion`, but the practically similar `safeguarding_form` validates on `blank`?

Reply:
Ordinary inconsistency — inclusion is better though, I think?

Reply:
I mean, `safeguarding_form` has to validate on `blank`, while `training_with_a_disability_form` has to validate on `inclusion`, what’s different behind the scenes in how these values are stored (just curious)?

Reply:
The disability form stores a boolean 

<img width="576" alt="Screenshot 2021-01-05 at 11 54 15" src="https://user-images.githubusercontent.com/642279/103643530-c4098880-4f4c-11eb-94c4-81dbeae5986c.png">

whereas the safeguarding form translates the answer to an enum

<img width="585" alt="Screenshot 2021-01-05 at 11 53 42" src="https://user-images.githubusercontent.com/642279/103643488-ad633180-4f4c-11eb-8f1b-d30cdab1fbc8.png">

This needn't make a difference to the validation logic, though — afaics it's still a required field and the acceptable answers are still 'yes' and 'no', I think?

Reply:
Ah-ha, thanks! I’ll try and pretend I never saw this inconsistency 😆 Validation logic is still the same, just that the translation string keys need to be different.